src/plotter.py

```diff
@@ -155,6 +155,68 @@ def make_table_plot_from_dict(self, this_dict,
         plt.close()
 
 
+    def make_sentiment_plot(self, this_dict,
+                            tosave=False):
+
+        # Get the data and labels
+        question = this_dict['question']
+        values   = this_dict['responses']['values']
```

Comment:
Would it be possible to auto-create bins for numerical histograms in the future (ie. salary ranges), or should we just make a new plotter function for that?

Reply:
This method is a bit specialized since it assumes certain things about the x axis categories. I tried to make the function work well specifically for the sentiment plots. 

I suggest we define a separate function for the more generic plotter.